### PR TITLE
Closes issue # 35 - View all tags

### PR DESCRIPTION
**Pull request by Ryan-L-Allred**
#35  is closed. The list of tags will now be displayed and a "Tags" link was added to the navbar.

### CODE REVIEW

File: SQL/02_Tabloid_Seed_Data.sql [hunk 1]
```diff
@@ -17,16 +17,16 @@ values (1, 'C#'), (2, 'JavaScript'), (3, 'Funny'), (4, 'Scary');
 set identity_insert [Tag] off
```
Comment (dalamcd) [inline SQL/02_Tabloid_Seed_Data.sql:17]: Please add this file to your .gitignore

File: Tabloid/Controllers/TagController.cs [hunk 2]
```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+using Tabloid.Models;
+using Microsoft.AspNetCore.Authorization;
+
+namespace Tabloid.Controllers
```
Comment (dalamcd) [inline Tabloid/Controllers/TagController.cs:7]: Users must be authorized to view data, make sure to add the [Authorize] attribute to this class or to each individual methods

File: Tabloid/Models/Tag.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
```
Comment (dalamcd) [inline Tabloid/Models/Tag.cs:4]: Don't need to include System.Runtime.InteropServices or System.Runtime.CompilerServices here

File: Tabloid/client/src/components/Tag.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      <p className="text-left px-2">{tag.name}</p>
+      <CardBody>
+        <iframe className="tag"
+          src={tag.url}
+          title="YouTube tag player"
+          frameBorder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowFullScreen />
+
+        <p>
```
Comment (dalamcd) [inline Tabloid/client/src/components/Tag.js]: The iframe code is left-over from what the video component used to display inline YouTube videos, it doesn't make any sense here in tags, please remove

File: Tabloid/client/src/components/TagList.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  };
+
+  useEffect(() => {
+    getTags();
+  }, []);
+
+  return (
+    <div>
+      {tags.map(t => 
+        <div>{t.name}</div>
```
Comment (dalamcd) [inline Tabloid/client/src/components/TagList.js:18]: This isn't using the Tag method from Tag.js and is instead just displaying the tag manually. If you'd like to do that, that's fine, but we can't have both. Either use the Tag component to display them or remove it, please.

**Review by dalamcd — CHANGES_REQUESTED** [review]

File: SQL/02_Tabloid_Seed_Data.sql (hunk 1 above)
Comment (Ryan-L-Allred) [reply SQL/02_Tabloid_Seed_Data.sql:17]: Do I just type the name of the file in the .gitignore?

**Review by dalamcd — APPROVED** [review]
Looks great, approved!

**Pull request merged**